docs/source/installation/vonBertalanffy.md

```diff
@@ -0,0 +1,125 @@
+# This is an instruction for installing all necessary tools to use vonBertalanffy
+
+## First, begin by installing all the necessary dependencies. At the time of writing this, the following were used:
+
+ * Ubuntu 16.04 LTS
+
+ * Python 2.7
+
+ * NumPy 1.11.1
+
+ * ChemAxon MarvinBeans 17.28.0
+
+ * OpenBabel 2.3
+
+The following commands are entered in a *terminal window (bash or similar  shell)*.
+
+### Python 2
+OpenBabel only works with Python 2. Most distributions should already have
+this installed, but if this is not the case, the following lines will do it:
```

Comment:
Add here that in order to run these commands, administrator rights are required.

Reply:
The commands have `sudo` - do you want me to comment that explicitly for the user?

Reply:
Yes, may you simply state that these commands required administrator rights? Thanks.

Reply:
Done